wagtailio/project_styleguide/templates/patterns/components/hero/hero.html

```diff
@@ -0,0 +1,27 @@
+<div class="grid">
+    <div class="hero">
+        <div class="hero__inner">
+            <h1 class="hero__heading">{{ value.heading }}</h1>
+
+            {% if value.subheading %}
+                <h2 class="hero__subheading intro-big">{{ value.subheading }}</h2>
+            {% endif %}
+
+            {% if value.intro %}
+                <p class="hero__intro">{{ value.intro }}</p>
+            {% endif %}
+
+            {% if value.cta_url %}
+                {% include "patterns/components/buttons/button.html" with arrow=True url=value.cta_url title=value.cta_text classes="hero__button" %}
+            {% elif value.cta_page %}
+                {% include "patterns/components/buttons/button.html" with arrow=True url=value.cta_page.url title=value.cta_text classes="hero__button" %}
+            {% endif %}
```

Comment:
Suggestion: As we're reusing the same variables but just changing the URL, could we do; 
```suggestion
            {% if value.cta_url or value.cta_page %}
                {% firstof value.cta_url value.cta_page.url as url %}
                {% include "patterns/components/buttons/button.html" with arrow=True url=url title=value.cta_text classes="hero__button" %}
            {% endif %}
```

Reply:
Nice, thanks